socorro/processor/cache_manager.py

```diff
@@ -50,7 +50,7 @@
 def count_sentry_scrub_error(msg):
     # NOTE(willkg): we re-use the processor prefix here and differentiate with the
     # service tag.
```

Comment:
seems like this comment is no longer relevant

Reply:
Good catch!